tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	parser "github.com/kubevirt/monitoring/pkg/metrics/parser"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+var excludedMetrics = map[string]struct{}{
+	"kubevirt_ssp_operator_up_total":           {},
+	"kubevirt_ssp_template_validator_up_total": {},
+}
+
+func readMetrics() []*dto.MetricFamily {
+	var metricFamilies []*dto.MetricFamily
+	metrics := metrics.RecordRulesDescList
```

Comment:
change the name to sspMetrics instead of metrics

Reply:
I used this name format for all linter pr's, so I think it's better to save this format, I think some operators use and expose also metrics from external locations.

Reply:
For me if I'm a developer that want to understand what we did or implement the linter, It will be easier to follow the references if they in the same format as possible, for example in [cdi operator](https://github.com/kubevirt/containerized-data-importer/pull/2774/files#diff-42fb4e8a81bc9dcc577c82ea3b473f3df7b7084a7277cf6e71b3ce2d5ba2dd94R57-R61) we use both `cdiMetrics` and `metricsList` so `metrics might be confusing to compere..